pt-BR/mail/chrome/messenger/messengercompose/composeMsgs.properties

```diff
@@ -86,7 +86,7 @@
 12517=Houve um erro ao enviar mensagens.\nResposta do servidor: “%s”.\nPor favor, verifique a mensagem e tente de novo.
 
 ## @name NS_ERROR_POST_FAILED
-12518=A mensagem não pôde ser postada porque houve falha na conexão ao servidor de news. O servidor pode estar indisponível ou recusando conexões. Por favor, verifique se sua configuração de news está correta e tente de novo ou contate o administrador da rede.
+12518=A mensagem não pôde ser postada porque houve falha na conexão ao servidor de notícias. O servidor pode estar indisponível ou recusando conexões. Por favor, verifique se sua configuração de news está correta e tente de novo ou contate o administrador da rede.
```

Comment:
CORRECTIONS NEEDED it should be 12518=A mensagem não pôde ser postada porque houve falha na conexão ao servidor de notícias. O servidor pode estar indisponível ou recusando conexões. Por favor, verifique se sua configuração de notícias está correta e tente de novo ou contate o administrador da rede.